### ISSUE TITLE
Improve the locking / unlocking accessibility when using screen readers

### ISSUE THREAD

**Issue by marvellousmachine** [NoobishSVK/fm-dx-webserver#52]
1. Currently it is not possible to lock / unlock server from the main screen when using a screen reader. You could either make it a HTML button as planned, or use the ARIA role button. When you go for the HTML button version, I suggest you would make the label dynamic: "Lock server" when unlocked and "Unlock server" when locked. 
2. Also it would be helpful if the window title would change in locked state by prepending something like 🔒 or 🔑. For example: "🔒 FM-DX Webserver [Server Name]"

**Comment by NoobishSVK** [comment]
1. I have added an aria-label (which will be available with the next update). 
2. The icons are already available, but were missing aria-labels, I have added them.

**Comment by NoobishSVK** [comment]
Button roles already seem to be there, I will fix the typo.

**Comment by NoobishSVK** [comment]
Please check now.

**Comment by marvellousmachine** [comment]
Perfect now. Thank you.
